src/features/course-main/utils/set-label.ts

```diff
@@ -0,0 +1,24 @@
+import dayjs from 'dayjs';
+import { hasDayInDate } from '.';
+import { Labels } from '@/app/types';
+
+export function setLabel(date: string) {
```

Comment:
getCourseStatus

Reply:
Please specify the return type explicitly

Reply:
And I'd a comment explaining the difference between PLANNED, AVAILABLE, and UPCOMING

Reply:
Done